### PR TITLE
EXP: Debug RTD maybe

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is to investigate #15695

- [ ] By checking this box, the PR author has requested that maintainers do **NOT** use the "Squash and Merge" button. Maintainers should respect this when possible; however, the final decision is at the discretion of the maintainer that merges the PR.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -96,10 +96,9 @@ docs = [
     "astropy[recommended]",  # installs the [recommended] dependencies
     "sphinx",
     "sphinx-astropy[confv2]>=1.9.1",
-    "pytest>=7.0",
```
Comment (pllim) [inline pyproject.toml:99]: Why do we even have pytest as "docs" dependency?!

**Comment by pllim** [comment]
Well, not these. Still no warning rendered. Will have to revisit next week; maybe try to stop using mambaforge and just use pip and see.

**Comment by ayshih** [comment]
As a reminder, we use conda on RTD for easy access to a non-bugged version of graphviz that will render the inheritance diagrams properly.  If using pip somehow fixes the warnings, we'd have to solve the graphviz issue a different way.

**Comment by pllim** [comment]
@ayshih is the version here bugged? I don't remember.

https://pypi.org/project/graphviz/

**Comment by ayshih** [comment]
Those are the Python bindings, which can be installed via pip, but the issue is that the graphviz library itself, which can't be installed via pip.

**Comment by pllim** [comment]
Good news: This isn't necessary after all. It was RTD beta add-ons.

File: pyproject.toml (hunk 1 above)
Comment (pllim) [reply pyproject.toml:99]: I dug up https://github.com/astropy/astropy/pull/8974 for this. So should just leave it be, I guess.

**Pull request closed without merging**